openapi_gitbook.yaml

```diff
@@ -147,7 +147,7 @@ paths:
                 source_lang:
                   $ref: '#/components/schemas/SourceLanguage'
                 target_lang:
-                  $ref: '#/components/schemas/TargetLanguage'
```

Comment:
Why is this using a new list instead of `TargetLanguage` and adding `ZH-HANT` there?

Reply:
@JanEbbing in the openapi.yaml file, we have a separate TargetLanguageText list so that we can communicate to users that `AR` and `ZH-HANT` are supported for text translation but _not_ document translation. We did not have such a list in openapi_gitbook.yaml, so I decided to add one—this way, `AR` and `ZH-HANT` would show up in the list of supported target languages here: https://developers.deepl.com/docs/api-reference/translate/openapi-spec-for-text-translation